src/implementationGuides/loadCompiledIGs.ts

```diff
@@ -4,17 +4,22 @@
  */
 
 import path from 'path';
-import { existsSync, readFileSync } from 'fs';
+import { existsSync, PathLike, readFileSync } from 'fs';
 
-const COMPILED_IGS_DIRECTORY = 'compiledImplementationGuides';
+export const COMPILED_IGS_DIRECTORY = 'compiledImplementationGuides';
 
 // eslint-disable-next-line import/prefer-default-export
-export const loadImplementationGuides = (moduleName: string): any[] | undefined => {
-    const implementationGuidesPath = path.join(__dirname, '..', '..', COMPILED_IGS_DIRECTORY);
-    const searchIgsPath = path.join(implementationGuidesPath, `${moduleName}.json`);
+export const loadImplementationGuides = (
+    moduleName: string,
```

Comment:
Hmm I was initially confused what the moduleName was I wonder if `moduleToLoad` or `moduleFor` is better maybe at the very least add comments to this method

Reply:
I added jsdoc comments to this method

Reply:
I added jsdoc comments to this method